src/internal/api/components/list.go

```diff
@@ -7,12 +7,19 @@ package components
 import (
 	"net/http"
 
-	"github.com/defenseunicorns/zarf/src/config"
+	"github.com/defenseunicorns/zarf/src/config/lang"
 	"github.com/defenseunicorns/zarf/src/internal/api/common"
+	"github.com/defenseunicorns/zarf/src/internal/cluster"
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	"github.com/go-chi/chi/v5"
 )
 
 // ListDeployingComponents writes a list of packages that have been deployed to the connected cluster.
-func ListDeployingComponents(w http.ResponseWriter, _ *http.Request) {
-	deployingPackages := config.GetDeployingComponents()
-	common.WriteJSONResponse(w, deployingPackages, http.StatusOK)
+func ListDeployingComponents(w http.ResponseWriter, r *http.Request) {
```

Comment:
```suggestion
func ListDeployedComponents(w http.ResponseWriter, r *http.Request) {
```

Deployed makes more sense than deploying with the refactor